### PR TITLE
Fix build after segmentation merge.

### PR DESCRIPTION
**Pull request by tdaede**

### CODE REVIEW

File: src/rdo.rs [hunk 1]
```diff
@@ -907,7 +907,7 @@ pub fn rdo_partition_decision(
                   let is_compound = ref_frames[1] != NONE_FRAME;
                   let mode_context = cw.find_mvrefs(bo, ref_frames, &mut mv_stack, subsize, false, fi, is_compound);
 
-                  cdef_coded = encode_block_a(seq, cw, if cdef_coded  {w_post_cdef} else {w_pre_cdef},
+                  cdef_coded = encode_block_a(seq, fs, cw, if cdef_coded  {w_post_cdef} else {w_pre_cdef},
```
Comment (ycho) [inline src/rdo.rs]: I wonder where new parameter 'fs' has added to this function?
Since one commit before (by me), c27a56946f, does not have it.

Comment (ycho) [reply src/rdo.rs]: Oh, apparently I read : 
pub fn encode_block_a(seq: &Sequence, fs: &FrameState,
cw: &mut ContextWriter ...
in src/encoder.rs

Then, how come my previous commit survived...?!

Comment (ycho) [reply src/rdo.rs]: Stupid I am, atomnuker's most recent patch changed it...!

**Comment by tdaede** [comment]
It was added by the segmentation commit.

**Pull request merged**